HEC.FDA.ViewModel/Saving/SavingBase.cs

```diff
@@ -289,15 +293,15 @@ public void DeleteRowWithKey( int key, string keyColName)
             SQLiteCommand command = Connection.Instance.Reader.DbConnection.CreateCommand();
             command.CommandText = sb.ToString();
             command.ExecuteNonQuery();
-        }             
+        }
 
         #endregion
 
         public int GetNextAvailableId()
         {
             //make sure the table exists
             OpenConnection();
-            DatabaseManager.DataTableView tbl = Connection.Instance.GetTable(_TableName);
+            object tbl = Connection.Instance.GetTable(_TableName);
```

Comment:
This is weird. If we're just confirming a table exists, can we do that with a bool instead of returning an object?

Reply:
yes, replaced with a bool wrapper function